solidrun-n6g.coffee

```diff
@@ -0,0 +1,29 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+module.exports =
+	version: 1
+	slug: 'n6g'
+	name: 'SolidSense N6G'
+	arch: 'armv7hf'
+	state: 'new'
+
+	supportsBlink: false
+
+	options: [ networkOptions.group]
+
+	yocto:
+		machine: 'solidrun-n6g'
+		image: 'balena-image'
+		fstype: 'balenaos-img'
+		version: 'yocto-dunsfell'
+		deployArtifact: 'balena-image-solidrun-n6g.balenaos.img'
```

Comment:
So this would run off an SD card? Does it not have internal storage to boot balenaOS from?